subsys/net/ip/ipv4.c

```diff
@@ -132,15 +132,13 @@ int net_ipv4_finalize_new(struct net_pkt *pkt, u8_t next_header_proto)
 			return net_udp_finalize(pkt);
```

Comment:
Here, the commit title "net/ipv4: Always return from the last finalizer" is also not very clear (as if the alternative was to infinite loop in it, or what?). Better: "net/ipv4: Always return **result of** the last finalizer"

Reply:
would sound better yes